### PR TITLE
Compile on template change, not every request

### PR DESCRIPTION
**Pull request by rulio**

### CODE REVIEW

File: epistream.coffee [hunk 1]
```diff
@@ -52,6 +52,15 @@ log.debug "%j", config
 log.debug "node version", process.version
 
 if config.isDevelopmentMode()
+  watch               = require('node-watch');
+  templates   = require './src//templates.coffee'
+  transformer = require './src/transformer.coffee'
+  recompile  = () -> 
+    console.log('TEMPLATES HAVE BEEN UPDATED -  RECOMPILING...')
```
Comment (igroff) [inline epistream.coffee:59]: You should use the log module as the rest of the log entries are doing

File: epistream.coffee [hunk 2]
```diff
@@ -52,6 +52,15 @@ log.debug "%j", config
...
+    templates.init()
+    transformer.init()
+
+  watch(config.templateDirectory, { recursive: true }, recompile );
   log.warn "********************************************************************************"
   log.warn "epiquery is running in development mode, this will result in templates not being cached and thus"
   log.warn "reloaded on every request, this will BE SLOW"
```
Comment (fearphage) [inline epistream.coffee:66]: Update this message to be truthful.

File: package.json [hunk 3]
```diff
@@ -37,6 +37,7 @@
     "lodash-contrib": "241.4.14",
     "moment": "2.19.3",
     "mysql": "2.18.1",
+    "node-watch": "^0.7.1",
```
Comment (igroff) [inline package.json:40]: Pin the version, and move the dependency to dev dependencies.

**Review by igroff — CHANGES_REQUESTED** [review]
Make those few changes and I think we're good to go.

File: epistream.coffee (hunk 2 above)
Comment (rulio) [reply epistream.coffee]: I have updated the wording

File: epistream.coffee (hunk 1 above)
Comment (rulio) [reply epistream.coffee]: done

**Review by bkelley123 — APPROVED** [review]

**Pull request merged**